docs/Templates/SpikeOutcome-Template.md

```diff
@@ -2,53 +2,40 @@
 
 ==================
 
-**Spike:** Spike_No
+**Spike:** Splashkit_camera
 
-**Title:** Spike_Title
+**Title:** Test Python Version
 
-**Author:** John Doe, john.doe@example.com
+**Author:** Darren Marchiano Sunandar
 
-## Goals / Deliverables
-
-Summarise from the spike plan goal* Besides this report, what else was created ie UML, code,
-reports*
-
-- Code see /spikes/spike04/
-- Short Report titled “IDE Comparison”
 
 ## Technologies, Tools, and Resources used
 
-List of information needed by someone trying to reproduce this work\
-
 - Internet Browser; Google Chroame, FireFox, Safari
 - Programming Languages:
-- Ruby
+- Python
 - Programming Libraries:
-- Minitest
+- Python libraries
 - Text Editor: VsCode
 - Terminal
 
 ## Tasks undertaken
 
 List key tasks likely to help another developer
 
-- ...
+- I've tested all the python function from the Splashkit website guides such as Animations, Audio, Camera, Graphics, etc. And there is 1 error,
```

Comment:
We need to list what has been tested and what has not been tested, preferably the actual functions.

Right now, not all of these guides have Python code in them, so we need to make a note of these ones.

We also need this like and not just finish with 'etc' because the tutorials are currently being produced, the next person could come along and assume that a tutorial added tomorrow has already been tested. We need to be specific.